src/main/java/com/google/sps/servlets/CronServlet.java

```diff
@@ -0,0 +1,157 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlet;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.GoodDeed;
+
+
+@WebServlet("/cronjob")
+public class CronServlet extends HttpServlet {
+    private static final String NAME = "Name";
+    private static final String DESCRIPTION = "Description";
+    private static final String POSTED_YET = "Posted Yet";
+    private static final String TIME_STAMP = "Timestamp";
+    private static final String DAILY_DEED = "Daily Deed";
+    private static final String GOOD_DEED = "GoodDeed";
+    private static final String FALSE = "false";
+    private static final String TRUE = "true";
+    private static final int MINIMUM_QUERY_LENGTH = 1;
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        // Clears the Daily Deed Property for all elements
+        resetDailyDeed();
+
+        // Pulls all Database Entries
+        List<GoodDeed> GoodDeeds = PullDeedsFromDatastore();
+
+        // Filters out Posted Deeds
+        List<GoodDeed> GoodDeeds_cleaned = cleanDeeds(GoodDeeds);
+
+        if (GoodDeeds_cleaned.size() < MINIMUM_QUERY_LENGTH) {
+            System.out.println("Query is below minimum size.");
+            
+            // Resets Posted Yet property of all posted deeds
+            resetPostedYet();
+            GoodDeeds_cleaned = GoodDeeds;
+            //System.out.println(GoodDeeds_cleaned);
```

Comment:
Remove this line

Reply:
@ShreeragJ the System.out.print line?

Reply:
Yes

Reply:
@ShreeragJ Got it. I removed all of the debugging print statements